scripts/installscripts/buster-install-mopidy3.sh

```diff
@@ -465,17 +465,23 @@ esac
 # power management of wifi: switch off to avoid disconnecting
 sudo iw dev wlan0 set power_save off
 
+# create backup of /etc/resolv.conf
+sudo cp /etc/resolv.conf /etc/resolv.conf.orig
+
 # Install required packages
 sudo apt-get update
 
 # some packages are only available on raspberry pi's but not on test docker containers running on x86_64 machines
 if [[ $(uname -m) =~ ^armv.+$ ]]
 then
-    sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages raspberrypi-kernel-headers
+    sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages install raspberrypi-kernel-headers
```

Comment:
pip should be python3?

Reply:
your comment is confusing me - there is no pip nor python in this line.